ibc-clients/ics08-wasm/types/src/lib.rs

```diff
@@ -21,8 +21,29 @@ pub mod msgs;
 #[cfg(feature = "cosmwasm")]
 pub mod serializer;
 
+use core::str::FromStr;
+
+use ibc_core_host_types::identifiers::ClientType;
 use ibc_primitives::prelude::Vec;
 pub type Bytes = Vec<u8>;
 
 pub static SUBJECT_PREFIX: &[u8] = b"subject/";
 pub static SUBSTITUTE_PREFIX: &[u8] = b"substitute/";
+
+pub const WASM_CLIENT_TYPE: &str = "08-wasm";
+
+/// Returns the wasm `ClientType`
+pub fn client_type() -> ClientType {
+    ClientType::from_str(WASM_CLIENT_TYPE).expect("Never fails because it's valid")
+}
+
+#[cfg(test)]
```

Comment:
This test is all about making sure we nail the `ClientType` construction. Just like we've got tests e.g. for `ChainId`, let's keep adding and maintaining such tests in their crates. Therefore, it is not necessary to include such tests in every new crate. wdyt?

Reply:
on a second thought, we need to make sure `WASM_CLIENT_TYPE` is a valid client type. eg. `8-wasm` is not a valid client type, but `08-wasm` is.

I will add some tests near `ClientType`, but I will leave this test alone, as this makes sure, in future, we maintain a valid `WASM_CLIENT_TYPE`.